plugins/ptp_operator/ptp_operator_plugin.go

```diff
@@ -256,7 +264,10 @@ func Start(wg *sync.WaitGroup, configuration *common.SCConfiguration, fn func(e
 		return nil
 	}
 	log.Infof("setting up status listener")
```

Comment:
update log message to: `setting up status listener for each publisher type`

Reply:
fixed